### PR TITLE
Ratings - change request code + updates to server/ form 

### PR DESCRIPTION
**Pull request by ryanaolea**
This is the code Wisdom and I wrote for call to api/interactions. Tested and edge cases are now being handled. Also includes a few fixes to server for potential use with postman. (res codes etc.)
Also cleaned up a little css tech debt and fixed issue that was hiding upload image buttons.

### CODE REVIEW

**Comment by ryanaolea** [comment]
delt with all the merge conflicts from yalls work last night and added the localhost to app for the interactions api call. if anyone has time to code review and merge this i can get to @FlightfulKiwi PR and we can have all our updates posted live!

File: client/src/App.jsx [hunk 1]
```diff
@@ -52,15 +64,23 @@ class App extends React.Component {
...
-        module: findId(e.target.parentNode),
-        element: e.target.outerHTML
+        element: e.target.outerHTML,
+        widget: findId(element)
       };
-      console.log('this is your click event', body);
 
-      // create axios post to /interactions here
+      axios.post(`${localhost}/interactions`, body)
+        .then(() => console.log('posted to interactions'))
```
Comment (FlightfulKiwi) [inline client/src/App.jsx]: Hey! Don't think this console log is needed anymore, yeah! I liked the one showing the component being clicked on and its details. Even that one isn't needed.

**Review by FlightfulKiwi — COMMENTED** [review]
One comment to address: not a world-ender. Just removing a console log.

File: client/src/App.jsx (hunk 1 above)
Comment (ryanaolea) [reply client/src/App.jsx]: ✅

**Review by FlightfulKiwi — APPROVED** [review]

**Pull request merged**